src/WebJobs.Script.WebHost/WebScriptHostManager.cs

```diff
@@ -349,6 +349,14 @@ protected override void OnHostStarted()
             _secretManager.PurgeOldFiles(Instance.ScriptConfig.RootScriptPath, Instance.TraceWriter);
         }
 
+        protected override void OnTimeout()
+        {
+            base.OnTimeout();
+            Stop();
+            Task.Delay(2000).Wait();
```

Comment:
I just made this async to make it easier.

---
In reply to: [75974974](https://github.com/Azure/azure-webjobs-sdk-script/pull/604#discussion_r75974974) [](ancestors = 75974974)